fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadJob.java

```diff
@@ -960,6 +960,9 @@ private void executeBeforeCheck(TransactionState txnState, TransactionStatus tra
     public void afterCommitted(TransactionState txnState, boolean txnOperated) throws UserException {
         long taskBeId = -1L;
         try {
+            // Update the job state is the job is too slow.
+            updateSubstate();
```

Comment:
original logic: the write lock is locked in `executeBeforeCheck` function and unlocked in this function at the end.
there will be some problems if unlock the write lock in `updateSubstate`.

Reply:
same in `afterAborted`